src/stories/Library/spacing/spacing.scss

```diff
@@ -12,6 +13,7 @@ $s-215: 215;
 
 $s-xs: 4px;
 $s-sm: 8px;
+$s-smd: 10px;
```

Comment:
Consider if we should ask Dept whether this 10px size is intentional. We use a 4/8-based size system where 10 does not really fit in.

Reply:
I think we could even go with a 12px option (8+4) - but there really is nothing between 8 and 16 pixels right now.
But I will ask in Figma :)